src/api/article/article.model.js

```diff
@@ -43,6 +43,9 @@ const ArticleSchema = new Schema({
     type: Boolean,
     default: true,
   },
+  pictureURL: String,
+  // This is a YouTube video ID. Ignored unless Article type is video.
+  videoID: String,
```

Comment:
imho better use camelCase - `videoId`

Reply:
debatable. In many case it's ok to use uppercase for such things like ID or URL